dti-website/src/data/all-projects.json

```diff
@@ -267,148 +91,188 @@
+    "heroEndingColor": "#F0F0F0",
+    "heroUseDarkText": true,
     "features": [
       {
-        "title": "Simple, But Bold.",
-        "description": "All of our feature pages include an autoplaying video to take users inside what it’s like to be a DTI member. Each of our pages follows a consistent design language that conveys the content of the page at a glance. Between each of these features, users are drawn to scroll and experience more of the website and find out more about our team."
+        "title": "Admin Web",
+        "description": "CULift riders can schedule and track their rides.",
+        "image": "/static/projects/carriage-feature1.png"
       },
       {
-        "title": "Showcasing Diversity.",
-        "description": "We especially care about having a diverse team full of unique perspectives, and we wanted to highlight our commitment on our Team page. View our statistics at a glance with our interactive chart visual."
+        "title": "Driver Mobile.",
+        "description": "This application facilitates communication between drivers and riders and also condenses drivers' busy schedules in an easy to read format that auto-updates as changes are made.",
+        "image": "/static/projects/carriage-feature2.png"
       },
       {
-        "title": "Apply Online.",
-        "description": "One of our biggest requests was a dedicated application page for students applying to our team. We made it easy for all interested applicants to find out about the recruitment process year-round."
+        "title": "Rider Web and Mobile",
+        "description": "Carriage’s rider platform allows riders to easily schedule and track their drivers through a seamless user interface.",
+        "image": "/static/projects/carriage-feature3.png"
       }
     ]
   },
   {
-    "name": "Zing",
-    "teamId": "zing",
-    "card": "/static/projects/zing-card.png",
-    "header": "Zing",
-    "subheader": "Study tools to help professors help students.",
+    "name": "CU Apartments",
+    "teamId": "apartments",
+    "card": "/static/projects/apartments-card.png",
+    "header": "CU Apartments",
+    "subheader": "Search for off-campus housing, review apartments, and share feedback!",
     "hero": {
-      "header": "What We Do",
-      "image": "",
-      "subheader": "Zing is an all-in-one web interface for professors to create optimal groups for class projects based on students’ study preferences. Built with the GroupEng algorithm developed by the Director of Cornell Engineering’s MTEI program, Zing enables professors to customize grouping criteria and easily match students. Our team also supports the Learning Strategies Center’s Study Partner program as a companion tool. Together, Zing helps student succeed inside and outside the classroom."
+      "header": "What we do",
+      "image": "/static/projects/apartments-header.png",
+      "subheader": "Search for off-campus housing, review apartments, and share feedback!"
```

Comment:
This is probably just what you were given by CU Apts but it's a bit odd that the "What we do" section and the general subheader are both "Search for off-campus housing, review apartments, and share feedback!". Wonder if we should make them different because it's awkward to read right now.